environments/prod/locals.tf

```diff
@@ -6,6 +6,7 @@
 
 locals {
   organization_config = data.terraform_remote_state.bootstrap.outputs.organization_config
+  net-host-prj = module.net-host-prj
```

Comment:
was already in the example branch - no change locally but needed in main
  net-host-prj = module.net-host-prj